kitsune/wiki/templates/wiki/edit.html

```diff
@@ -22,6 +22,12 @@
 
     <article id="edit-document" {% if locked %}class="locked"{% endif %} data-slug="{{ document.slug }}">
       <h1>{{ _('<em>Editing</em> {title}')|fe(title=document.title) }}</h1>
+      {% if document_needs_change %}
+        <p>
+          {% trans comment=_(document.needs_change_comment) %} <b>Needs Change:</b> {{comment}} {% endtrans %}
```

Comment:
Don't put the needs_change comment into `_`. `_` is only for translating static strings (or very special database driven variables with special tags). Even if it did need translated, you are translating it twice here, once with `_` and again with `{% trans ... %}`.